package.json

```diff
@@ -28,7 +32,9 @@
     "asbuild:uastc-astc": "asc assembly/uastc-astc.ts -b build/uastc_astc.wasm -t build/uastc_astc.wat --optimize --converge --runtime none --noAssert --importMemory",
     "asbuild:uastc-bc7": "asc assembly/uastc-bc7.ts -b build/uastc_bc7.wasm -t build/uastc_bc7.wat --optimize --converge --runtime none --noAssert --importMemory",
     "asbuild:transcoders": "npm run asbuild:uastc-bc7 && npm run asbuild:uastc-astc && npm run asbuild:uastc-rgba32-unorm && npm run asbuild:uastc-rgba32-srgb",
-    "asbuild": "npm run asbuild:generator && npm run asbuild:transcoders"
+    "asbuild": "npm run asbuild:generator && npm run asbuild:transcoders",
+    "test": "ts-node node_modules/tape/bin/tape test/test.ts | tap-spec",
+    "test:ci": "CI=1 npm test"
```

Comment:
This command doesn't work on Windows (it needs to be like `set CI=1`). If there's no easy way to make this portable, we could keep it as is.

Reply:
Good to know. The CI doesn't actually need this macro — its env variable is configured by the `.yml` file — so let's just omit it and we can use the OS-specific commands as needed.